assets/queries/openAPI/parameter_object_undefined_type/query.rego

```diff
@@ -0,0 +1,24 @@
+package Cx
+
+import data.generic.openapi as openapi_lib
+
+CxPolicy[result] {
+	doc := input.document[i]
+	openapi_lib.check_openapi(doc) != "undefined"
+	params := doc.paths[name].parameters[n]
```

Comment:
Parameters can be defined in other attributes - https://swagger.io/docs/specification/describing-parameters/#path-parameters. For example, path[name].get.parameters